build.gradle

```diff
@@ -771,15 +774,20 @@ jlink {
         requires 'jakarta.xml.bind'
         requires 'org.apache.commons.lang3'
         requires 'org.apache.commons.text'
-        requires 'org.apache.commons.logging';
+        requires 'org.apache.commons.logging'
```

Comment:
You need to add the providers as well
Uses always required the provides if the dependency is not modularized

Reply:
Note that this change comes from the reordering. Did not see any exceptions while logging the last months.